### PR TITLE
BL-5898 – Fix concerns from first DAC assessment

### PR DESCRIPTION
**Pull request by mathieuhendey**

### CODE REVIEW

File: webapp/src/main/java/uk/gov/dvsa/motr/remote/vehicledetails/VehicleDetails.java [hunk 1]
```diff
@@ -17,9 +17,6 @@
     @JsonProperty("model")
     private String model;
 
-    @JsonProperty("makeInFull")
```
Comment (aaronlong) [inline webapp/src/main/java/uk/gov/dvsa/motr/remote/vehicledetails/VehicleDetails.java]: It looks like this merge request might be reverting the changes made for the make in full story: https://github.com/dvsa/motr-webapp/pull/150/files

File: webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/service/SmsConfirmationService.java [hunk 2]
```diff
@@ -111,8 +111,7 @@ public Confirmation verifySmsConfirmationCode(String vrm, String phoneNumber, St
 
         if (smsConfirmation.getCode().equals(confirmationCode)
                 && smsConfirmation.getPhoneNumber().equals(phoneNumber)
-                && smsConfirmation.getVrm().equals(vrm)
-                && smsConfirmation.getAttempts() < MAX_ATTEMPTS) {
+                && smsConfirmation.getVrm().equals(vrm)) {
```
Comment (aaronlong) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/service/SmsConfirmationService.java]: Again looks like this branch might be reverting some other stories.  Need to look into this further / rebase

File: webapp/src/main/java/uk/gov/dvsa/motr/web/resource/ReviewResource.java [hunk 3]
```diff
@@ -104,7 +104,6 @@ public Response reviewPage() throws Exception {
                     .setExpiryDate(vehicle.getMotExpiryDate())
                     .setMake(vehicle.getMake())
                     .setModel(vehicle.getModel())
-                    .setMakeInFull(vehicle.getMakeInFull())
```
Comment (aaronlong) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/resource/ReviewResource.java]: This change needs to be reverted to allow the ViewModel to set MakeInFull

Comment (wkainos) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/resource/ReviewResource.java]: Any reason for this?

Comment (mathieuhendey) [reply webapp/src/main/java/uk/gov/dvsa/motr/web/resource/ReviewResource.java]: @wkainos bad rebase, fixing now

**Pull request merged**